.github/CHANGELOG.md

```diff
@@ -73,6 +76,10 @@ Lee J. O'Riordan.
 
 ### Improvements
 
+* `lightning.gpu` is decoupled from Numpy layer during initialization and execution
+and change lightning.gpu to inherit from QubitDevice instead of LightningQubit.
+[(#70)](https://github.com/PennyLaneAI/pennylane-lightning-gpu/pull/70)
+
```

Comment:
I believe this log entry should be transferred to the right release, or even merged with the one already there.

Reply:
will do when merge main into this PR.

Reply:
done